Curtains/Curtains.Application/DTO/ReviewDTO.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AutoMapper;
+using AutoMapper.Configuration.Annotations;
+using Curtains.Domain.Models;
+
+namespace Curtains.Application.DTO
+{
+
+    /// <summary>
+    /// Class <c> ReviewDTO </c> describes interaction model for ReviewController and database.
+    /// </summary>
+    public class ReviewDTO
+    {
+        #region PropertiesRegion
+        /// <summary>
+        /// This property represents unique identifier for data tranfer objects.
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// This property represents title of data transfer objects.
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// This property represents description of data transfer objects.
+        /// </summary>
+        public string Description { get; set; }
```

Comment:
We need to do something with this kludge cuz there is no Description and Title in Review table in database

Reply:
now or later

Reply:
Maybr we can use this properties later ?

Reply:
Maybe we just should remove Title and Description fields from DTO?

Reply:
Deleted